### ISSUE TITLE
Optimize dynamic load of CSS blocks

### ISSUE THREAD

**Issue by htmgarcia** [publishpress/PublishPress-Blocks#484]
The CSS from other plugin's blocks is outputted in body, so our plugin may crawl that CSS and merge into "advgb-styles-renderer" `<style>` tag.

Let's find a way to load only our CSS but without depending on a registered style that won't load always as happened prior to implementing this fix https://github.com/publishpress/PublishPress-Blocks/commit/91105d15c3a4f2e8d5cee227435b912e1e6ba8c8#diff-48138e99a9c02e19da943fd0142428c28ef5342ad8e32f1b5554ac10d706a25b

In other words, using wp_add_inline_style() is a bad idea due 'advgb_blocks_styles' (blocks.css file) loads only on pages/posts with PP Blocks. 
_____
Related to #476
